libraries/helpers.rb

```diff
@@ -44,6 +44,26 @@ def chef_vault_item(bag, id)
     end
   end
 
+  # Helper method that allows for listing the ids of a vault in a recipe.
+  # This method is needed because data_bag() returns the keys along with
+  # the items, so this method strips out the keys for users so that they
+  # don't have to do it in their recipes.
+  # @example
+  # ids = chef_vault('secrets')
+  # log 'Yeah buddy!' if ids[0] == 'bacon'
+  # @param [String] bag Name of the data bag to load from.
+  # @return [Array]
+  def chef_vault(bag)
+    items = []
+    raise "'#{bag}' is not a vault" unless Chef::DataBag.list.include? bag
+    data_bag(bag).each do |id|
```

Comment:
A more idiomatic way of writing this would be:
```
def chef_vault(bag)
  raise "'#{bag}' is not a vault" unless Chef::DataBag.list.include? bag
  pattern = Regexp.new(/_keys$/).freeze
  data_bag(bag).each_with_object([]) do |id, acc|
    acc << id unless pattern.match?(id)
  end
end
```